### ISSUE TITLE
PDF Download

### ISSUE THREAD

**Issue by peterhunn** [accordproject/techdocs#96]
**Is your feature request related to a problem? Please describe.**

NA

**Describe the solution you'd like**

I'd like to be able to download the current version of the technical specification and associated documentation in a single PDF reference

**Comment by sesam** [comment]
What does "associated documentation" mean here? Does this issue include creating a PDF and adding a suitably visible link to it?

**Comment by irmerk** [comment]
@peterhunn flagging this for you.

**Comment by peterhunn** [comment]
> What does "associated documentation" mean here? Does this issue include creating a PDF and adding a suitably visible link to it?

The Template Spec is one component of the documentation (currently available at https://docs.accordproject.org/). 

This issue involves generating a downloadable PDF representation of all of AP documentation. 

N.B. As @dselman added, the documentation is flagged for significant revision under #111.

**Comment by mttrbrts** [comment]
One approach to solving this problem would be to find a solution to converting the Docusaurus markdown content to PDF. 
For example, https://github.com/facebook/docusaurus/issues/969

IMO the scope should only be the contents of https://github.com/accordproject/techdocs/tree/master/docs, ideally with a way to preserve the ordering / structure of the content in Docusaurus.

If this can be done in a general way, it should be independent of the ongoing changes to the documentation structure in #111.

**Comment by jeromesimeon** [comment]
I agree with @mttrbrts We should wait for that feature to be available in Docusaurus. Last time I checked it seemed to be a widely requested one, and (I think) slated for the pretty big Docusaurus 2.0 update.

**Comment by RobbiNespu** [comment]
Do you use travis? we can build PDF with pandoc during build

**Comment by dselman** [comment]
It seems like the Docusaurus community is making some progress on this. E.g.
https://github.com/KohheePeace/docusaurus-pdf
